sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystTypeConverters.scala

```diff
@@ -470,7 +478,10 @@ object CatalystTypeConverters {
    * This method is slow, and for batch conversion you should be using converter
    * produced by createToScalaConverter.
    */
-  def convertToScala(catalystValue: Any, dataType: DataType): Any = {
-    createToScalaConverter(dataType)(catalystValue)
+  def convertToScala(
+      catalystValue: Any,
+      dataType: DataType,
+      useJava8DateTimeApi: Boolean = SQLConf.get.datetimeJava8ApiEnabled): Any = {
```

Comment:
The only occurrence using this default value seems to be `toJson(CatalystTypeConverters.convertToScala(udt.serialize(v), dataType), dataType)`. Shall we remove this default value?

Reply:
Removed. Thanks.

Reply:
I made a little mistake. Let me revert the last commit, and just remove the default value.